portal/server/controllers/dashboard/filters/generate-template-filters.js

```diff
@@ -12,17 +12,7 @@ const { SUBMISSION_TYPE } = require('../../../constants');
  * @returns { text: 'GEF deals', value: 'GEF', checked: true }
```

Comment:
Could you update the params of this jsdoc please to be correct to the parameters below?